.github/workflows/release-nightly.yml

```diff
@@ -43,3 +44,36 @@ jobs:
           prerelease: true
           removeArtifacts: true
           tag: 'nightly'
+
+  release_pypi:
+    name: Make Release on pypi.anaconda.org
+    needs: build_wheels
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: write
```

Comment:
Is this permissions setting also needed here or is it only relevant for the step above (and was accidentally copy/pasted)?